lib/nerves_hub_link/extensions/logging/logger_handler.ex

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2025 Josh Kalderimis
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+defmodule NervesHubLink.Extensions.Logging.LoggerHandler do
+  @moduledoc """
+  Send logs to NervesHub.
+  """
+
```

Comment:
```suggestion
@behaviour :logger_handler
```
I think we should include the behavior here.

Reply:
Soooooo I added it, but its not available on Erlang versions under 27.x

Thoughts?